netlify.toml

```diff
@@ -1,2 +1,5 @@
 [build]
+  base = "client/flutter"
+  command = "chmod +x flutter-netlify-build.sh && ./flutter-netlify-build.sh"
```

Comment:
Executable bit can be set in the repo, no need to apply it here (afaik)

Reply:
you mean it can be set in Netlify? yes I think that's possible